docs/src/pages/components/badge/demo.tsx

```diff
@@ -1,21 +1,34 @@
-import React from 'react';
+import * as React from 'react';
 import { Badge, Flex } from '@aws-amplify/ui-react';
 
-import { BadgePropControls } from '@/components/BadgePropControls';
-import { useBadgeProps } from '@/components/useBadgeProps';
-import { Example } from '@/components/Example';
+import { BadgePropControls } from './BadgePropControls';
+import { useBadgeProps } from './useBadgeProps';
+import { Demo } from '@/components/Demo';
+
+const propsToCode = (badgeProps) => {
+  return (
+    `<Badge` +
+    (badgeProps.size ? `\n  size="${badgeProps.size}"` : '') +
```

Comment:
should we escape these values? Bad people could abuse props values to inject unexpected props to generated code.

Reply:
That is a good point.. do you think we need that for when the values come from a select field or just when they come from a text field?

Reply:
I feel we need to quote those values regardless of where the value come from. A simple quote/double quote escaping should be enough.